src/lib/viewers/archive/ArchiveExplorer.js

```diff
@@ -0,0 +1,161 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import memoize from 'lodash/memoize';
+import Internationalize from 'box-ui-elements/es/elements/common/Internationalize';
+import {
+    readableTimeCellRenderer,
+    sizeCellRenderer,
+    itemNameCellRenderer,
+} from 'box-ui-elements/es/features/virtualized-table-renderers';
+import VirtualizedTable from 'box-ui-elements/es/features/virtualized-table';
+import { Column } from 'react-virtualized/dist/es/Table/index';
+import { TABLE_COLUMNS } from './constants';
+
+const { KEY_NAME, KEY_MODIFIED_AT } = TABLE_COLUMNS;
```

Comment:
I don't think destructuring is necessary here.

Reply:
why? isn't `TABLE_COLUMNS. KEY_NAME` too long?

Reply:
> I don't think destructuring is necessary here.

why? isn't `TABLE_COLUMNS. KEY_NAME` too long?